doc/introduction/inspecting_circuits.rst

```diff
@@ -128,20 +133,25 @@ Currently supported devices include:
 * ``default.mixed``: each snapshot saves the density matrix
 * ``default.gaussian``: each snapshot saves the covariance matrix and vector of means
 
-During normal execution, the snapshots are ignored:
```

Comment:
Moved this sentence below, to an actual execution of the circuit that is related to the sentence.